### PR TITLE
Added edit metadata GUI elements

### LINKED ISSUE #2455: crowdsourcing Metadata
Allow any user to improve the metadata. Examples of existing approaches:

image

image

image

Time slicing is too heavy for Tribler, out of scope. Just metadata.
image

### PR DESCRIPTION
**Pull request by devos50**
- Added 'edit metadata' GUI elements.
- Renamed various files and classes to reflect our new knowledge-based mechanism.

Closes #2455.

196442102-16ef039f-49b6-42b2-a4a0-8a6061cdedcb-2

### CODE REVIEW

**Comment by drew2a** [comment]
Maybe it is possible to add the Language Dropdown from a web toolkit and render it via the QT WebEngine widget?  
Like: https://coderthemes.com/ubold/layouts/default/dark.html

image

File: src/tribler/core/components/metadata_store/utils.py [hunk 1]
```diff
@@ -73,6 +73,17 @@ def _add_operation(_tag, _op, _key):
         _add_operation(tag, Operation.ADD, random_key_3)
         _add_operation(tag, Operation.REMOVE, random_key_2)
 
+    # Give the torrent some simple attributes
+    random_title = generate_title(2)
+    random_year = "%s" % random.randint(1990, 2040)
+    random_description = generate_title(5)
+    random_lang = random.choice(["english", "russian", "dutch", "klingon", "valyerian"])
```
Comment (drew2a) [inline src/tribler/core/components/metadata_store/utils.py:80]: Valyrio muño ēngos ñuhys issa.

**Comment by devos50** [comment]
I've added a drop-down menu for the language field:

Schermafbeelding 2022-10-31 om 10 49 34

Schermafbeelding 2022-10-31 om 10 49 30

These languages are source from [this JSON file](https://gist.github.com/Josantonius/b455e315bc7f790d14b136d61d9ae469), which we include in our bundle.

File: src/tribler/gui/qt_resources/languages.json [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+  "yi": "Yiddish",
+  "yo": "Yoruba",
+  "zh": "Chinese",
+  "zh-cn": "Chinese (China)",
+  "zh-hk": "Chinese (Hong Kong SAR)",
+  "zh-mo": "Chinese (Macau SAR)",
+  "zh-sg": "Chinese (Singapore)",
+  "zh-tw": "Chinese (Taiwan)",
+  "zu": "Zulu"
+}
```
Comment (drew2a) [inline src/tribler/gui/qt_resources/languages.json:220]: NIT: a new line :)

**Review by drew2a — APPROVED** [review]

File: src/tribler/gui/qt_resources/languages.json (hunk 2 above)
Comment (devos50) [reply src/tribler/gui/qt_resources/languages.json]: Added

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

**Pull request merged**